client/components/signup-form/index.jsx

```diff
@@ -447,6 +456,10 @@ export default React.createClass( {
 							{ this.props.formHeader }
 						</header>
 					}
+					{ this.props.isSocialSignupEnabled && <SocialSignupForm /> }
+					{ this.props.isSocialSignupEnabled && <HrWithText>
+						{ i18n.translate( 'Or sign up with your email address:' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **15** times:
`translate( 'Enter your sign up email address' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).